### ISSUE TITLE
Cannot open PE file due to paremeter error

### ISSUE THREAD

**Issue by TsXor** [andyjsmith/Exe-Spy#2]
Installation: `pip install exespy`
Execution: `exespy`

Error when opening PE file:
```
Traceback (most recent call last):
  File "D:\Python\3.11\Lib\site-packages\exespy\exespy.py", line 152, in show_open_file
    self.load_pe(file_selection[0])
  File "D:\Python\3.11\Lib\site-packages\exespy\exespy.py", line 192, in load_pe
    self.pe = pe_file.PEFile(path)
              ^^^^^^^^^^^^^^^^^^^^
  File "D:\Python\3.11\Lib\site-packages\exespy\pe_file.py", line 45, in __init__
    self.lief_obj = lief.parse(raw=self.data, name=self.name)
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: parse(): incompatible function arguments. The following argument types are supported:
    1. parse(raw: bytes) -> Optional[lief._lief.Binary]
    2. parse(filepath: str) -> Optional[lief._lief.Binary]
    3. parse(obj: Union[io.IOBase | os.PathLike]) -> Optional[lief._lief.Binary]

Invoked with types: kwargs = { raw: bytes, name: str }
```

Fixed with: (`exespy/pe_file.py`, L45)
```python
        #self.lief_obj = lief.parse(raw=self.data, name=self.name)
        self.lief_obj = lief.parse(raw=self.data)
```

**Comment by andyjsmith** [comment]
Thanks for letting me know! Fixed and uploaded to PyPI as v1.0.1

**Comment by sage936** [comment]
Hi, @andyjsmith . I'm seeing this with v1.0.1 on Windows 11:

```
Traceback (most recent call last):
  File "C:\Python313\Lib\site-packages\exespy\exespy.py", line 152, in show_open_file
    self.load_pe(file_selection[0])
    ~~~~~~~~~~~~^^^^^^^^^^^^^^^^^^^
  File "C:\Python313\Lib\site-packages\exespy\exespy.py", line 192, in load_pe
    self.pe = pe_file.PEFile(path)
              ~~~~~~~~~~~~~~^^^^^^
  File "C:\Python313\Lib\site-packages\exespy\pe_file.py", line 45, in __init__
    self.lief_obj = lief.parse(raw=self.data)
                    ~~~~~~~~~~^^^^^^^^^^^^^^^
TypeError: parse(): incompatible function arguments. The following argument types are supported:
    1. parse(obj: Union[str | io.IOBase | os.PathLike | bytes | list[int]]) -> lief._lief.PE.Binary | lief._lief.OAT.Binary | lief._lief.ELF.Binary | lief._lief.MachO.Binary | lief._lief.COFF.Binary | None

Invoked with types: kwargs = { raw: bytes }
```

Nothing will open currently.
